Blackfynn/data-portal
# File browser #188u4v

# Description

The purpose of this PR is to add the file browser to a dataset's page.

![127 0 0 1_5000_browse_(Laptop with MDPI screen) (1)](https://user-images.githubusercontent.com/1493195/62304480-0bccdc00-b44c-11e9-9a2d-088fa683e978.png)

## Type of change


# How Has This Been Tested?

- Open a dataset
- Scroll down to the Files section
- The "Name" column should be sorted, and sortable
- Click on a folder
- The contents of the folder should load, and the breadcrumbs should update
- Open another folder if applicable
- Use the breadcrumb to navigate
- Open Vue devtools and inspect `FilesTable`
- Set `hasError` to `true`
- Error state should display
- Click on the "Try again" button
- Data should display properly, and error state should be removed.

# Checklist:
